documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -866,6 +866,13 @@ following sections show you how you can achieve index time boosting using Hibern
 [[section-boost-annotation]]
 ==== Static index time boosting
 
+[WARNING]
+====
+Index-time boosting is deprecated: it will be removed starting from Lucene 7.
+
+To implement static boosting, you should use query-time boosting instead: see <<search-query>>>.
```

Comment:
"static" ? Why not simply say "We recommend to use query-time boosting instead : .." ?